### PR TITLE
Add TAC Meeting files for 2024-08-22

### PR DESCRIPTION
**Pull request by riaankleinhans**
/cc @bensternthal

### CODE REVIEW

File: TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - David Kaplan + / Harold Gilkey 
+* Arm - Nathaniel McCallum   / Michael Lu
+* Google - Catherine Zhang
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) +  / Simon Johnson
+* Meta -  Henry Wang /  Kevin Hui
+* Microsoft - Alec Fernandez
+* Nvidia - Fritz Alder
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md:44]:
```suggestion
* Nvidia - Fritz Alder +
```

File: TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+* Chandra Nelogal (Self)
+* David Kaplan (AMD)
+* Henk Birkholz
+* Jason Rogers (Invary)
+* Jens Albers (Fr0ntierX)
+* Jo Van Bulck
+* Joseph Artgole (ARM)
+* Julian Stephen (IBM)
+* Kanth Ghatraju (Oracle)
+* Krzystof Sandowizc
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md:62]:
```suggestion
* Krzysztof Sandowicz (Intel)
```

File: TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - David Kaplan + / Harold Gilkey 
+* Arm - Nathaniel McCallum   / Michael Lu
+* Google - Catherine Zhang
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md:39]:
```suggestion
* Google - Catherine Zhang +
```

File: TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) +  / Simon Johnson
+* Meta -  Henry Wang /  Kevin Hui
+* Microsoft - Alec Fernandez
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md:43]:
```suggestion
* Microsoft - Alec Fernandez +
```

**Review by mssun — APPROVED** [review]

File: TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+Ben Sternthal (BS) suggested that the group compile a list of asks and potential companies to approach for free credits. BS also mentioned the possibility of offering co-marketing opportunities to partners.
+
+## Budget
+
+DM asked committee members to start thinking about what they would like to see in the 2025 budget as that would be discussed in the upcoming TAC meetings.
+
+## Any Other Business / Schedule
+
+Next meeting August 22, 2024
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-08-22/CCC-TAC-Minutes-2024-08-22.md:121]:
```suggestion
Next meeting September 05, 2024
```
Or just delete this line as one less thing to update in each minutes.

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**